js/script.js

```diff
@@ -59,6 +82,26 @@
     }
   })
 
+  app.service("UserStorage", function($http, API){
+
+    this.user = null;
+
+    this.setUser = function(){
```

Comment:
I think in this method you should pass `user` param and assign it to internal var `user`